src/NormalizingFlows.jl

```diff
@@ -20,17 +19,17 @@ function NF(
     rng::AbstractRNG=Random.GLOBAL_RNG,
     max_iters::Int=1000,
     optimiser::Optimisers.AbstractRule=Optimisers.ADAM(),
-    AD_backend::AD.AbstractBackend=AD.ZygoteBackend(),
+    ADbackend::ADTypes.AbstractADType=ADTypes.AutoZygote(),
```

Comment:
> I would still weakly prefer Symbol instead of ADTypes for user-facing functions. I think this may increase the mental load of users somewhat, because it forces user to install and using the ADType package. My opinion is not very strong on this.
I agree on this; Symbol is clearly more aesthetically pleasing. However, ADTypes enables storing configurations of AD method in the object, e.g., chunksize ofor ForwardDiff, which I'm not sure how to realize via Symbols. I think so far, I'll keep the ADTypes; we can always change it easily later if we want.